gadget/mountedfilesystem.go

```diff
@@ -383,6 +378,16 @@ func (f *mountedFilesystemUpdater) Update() error {
 		}
 	}
 
+	knownContent := make(map[string]bool)
+	for _, c := range f.ps.VolumeStructure.Content {
+		dst := c.Target
+		if strings.HasSuffix(dst, "/") {
+			dst = filepath.Join(dst, filepath.Base(c.UnresolvedSource))
+		}
+		logger.Noticef("DEBUGME: known asset %s", dst)
```

Comment:
drop?

Reply:
Done. And there was a 3rd one as well I have removed.